circle.yml

```diff
@@ -61,6 +61,11 @@ test:
         parallel: true
     - codecov:
         parallel: true
-    - rm -rf docs/API && make -C docs clean
-    - python setup.py docs:
+    # sphinx fails with python 3.5.1 due to an issue in typing package,
+    # and node 1 has python 3.4.3
+    - if [ $CIRCLE_NODE_INDEX == 1 ] ; then pip install -r docs-requirements.txt ; fi:
+        parallel: true
+    - if [ $CIRCLE_NODE_INDEX == 1 ] ; then rm -rf docs/API && make -C docs clean ; fi:
```

Comment:
@jayvdb Removing `docs/API` is important to make autodoc always regenerate it from the python modules, what can lead to errors as we had them here... That should always be checked!